src/action.c

```diff
@@ -466,6 +515,30 @@ view_for_action(struct view *activator, struct server *server,
 	}
 }
 
+static void
+run_if_action(struct view *view, struct server *server, struct action *action)
+{
+	struct view_query *query;
+	struct wl_list *queries, *actions;
+	const char *branch = "then";
+
+	queries = action_get_qlist(action, "query");
+	if (queries) {
+		branch = "else";
+		wl_list_for_each(query, queries, link) {
+			if (view_matches_query(view, query)) {
+				branch = "then";
+				break;
+			}
+		}
+	}
```

Comment:
Suggest documenting the implicit `OR` logic between queries here, simple 1-line comment should be enough.

Reply:
Why not, done.